features/step_definitions/basic_steps.rb

```diff
@@ -279,6 +279,10 @@ def current_fullpath
   page.find(:css, '#sidebar')
 end
 
+Then(/^I should not see the sidebar$/) do
+  expect(page).to_not have_selector('#sidebar')
```

Comment:
@mattwr18 same same, don't assert sth. is not visible. At least check sth. that's there for sure and compare e.g. the html structure :man_shrugging: or maybe even consider not to write tests. Those tests are likely to provide little confidence but will give some extra work on refactorings.